packages/hardhat/contracts/strategy/CrabStrategy.sol

```diff
@@ -268,25 +282,21 @@ contract CrabStrategy is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Ownab
     /**
      * @notice deposit ETH into strategy
      * @dev provide ETH, return wSqueeth and strategy token
-     * @return wSqueethToMint minted amount of wSqueeth
-     * @return depositorCrabAmount minted amount of strategy token
      */
-    function deposit() external payable nonReentrant returns (uint256, uint256) {
+    function deposit() external payable nonReentrant {
         uint256 amount = msg.value;
 
         (uint256 wSqueethToMint, uint256 depositorCrabAmount) = _deposit(msg.sender, amount, false);
 
         emit Deposit(msg.sender, wSqueethToMint, depositorCrabAmount);
-
-        return (wSqueethToMint, depositorCrabAmount);
```

Comment:
I didn't see a reason to return this, is there a reason we would want to do it? We don't do anything similar for flashDeposit or any of the withdrawal functions.

Reply:
I was initially returning values in case the calling smart contract need them...

Reply:
Makes sense. It wouldn't be so useful unless all functions had it, right?

Smart contracts could also query balanceOf before and after to get the changes if they implemented deposits/withdrawals, etc?

Keep it removed or change back, thoughts?